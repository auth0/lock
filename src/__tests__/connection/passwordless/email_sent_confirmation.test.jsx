import React from 'react';
import I from 'immutable';

import { expectComponent } from 'testUtils';

import EmailSentConfirmation from '../../../connection/passwordless/email_sent_confirmation';

const lock = I.fromJS({ id: '__lock-id__' });

describe('EmailSentConfirmation', () => {
  it('renders correctly', async () => {
    expectComponent(<EmailSentConfirmation lock={lock} />).toMatchSnapshot();
  });
});
