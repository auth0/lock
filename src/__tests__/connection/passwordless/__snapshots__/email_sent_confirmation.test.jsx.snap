// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailSentConfirmation renders correctly 1`] = `
<div
  className="auth0-lock-confirmation"
>
  <span
    aria-label="back"
    className="auth0-lock-back-button"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<svg aria-hidden=\\"true\\" focusable=\\"false\\" enable-background=\\"new 0 0 24 24\\" version=\\"1.0\\" viewBox=\\"0 0 24 24\\" xml:space=\\"preserve\\" xmlns=\\"http://www.w3.org/2000/svg\\" xmlns:xlink=\\"http://www.w3.org/1999/xlink\\"> <polyline fill=\\"none\\" points=\\"12.5,21 3.5,12 12.5,3 \\" stroke=\\"#000000\\" stroke-miterlimit=\\"10\\" stroke-width=\\"2\\"></polyline> <line fill=\\"none\\" stroke=\\"#000000\\" stroke-miterlimit=\\"10\\" stroke-width=\\"2\\" x1=\\"22\\" x2=\\"3.5\\" y1=\\"12\\" y2=\\"12\\"></line> </svg>",
      }
    }
    id="__lock-id__-back-button"
    onClick={[Function]}
    onKeyPress={[Function]}
    role="button"
    tabIndex={0}
  />
  <div
    className="auth0-lock-confirmation-content"
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "<svg focusable=\\"false\\" width=\\"56px\\" height=\\"56px\\" viewBox=\\"0 0 52 52\\" version=\\"1.1\\" xmlns=\\"http://www.w3.org/2000/svg\\" xmlns:xlink=\\"http://www.w3.org/1999/xlink\\" class=\\"checkmark\\"> <circle cx=\\"26\\" cy=\\"26\\" r=\\"25\\" fill=\\"none\\" class=\\"checkmark__circle\\"></circle> <path fill=\\"none\\" d=\\"M14.1 27.2l7.1 7.2 16.7-16.8\\" class=\\"checkmark__check\\"></path> </svg>",
        }
      }
    />
    <p>
      <span
        dangerouslySetInnerHTML={
          Object {
            "__html": " ",
          }
        }
      />
    </p>
    <span>
      <a
        className="auth0-lock-resend-link"
        href="javascript:void(0)"
        onClick={[Function]}
      >
        
         
        <span
          dangerouslySetInnerHTML={
            Object {
              "__html": "<svg focusable=\\"false\\" height=\\"32px\\" style=\\"enable-background:new 0 0 32 32;\\" version=\\"1.1\\" viewBox=\\"0 0 32 32\\" width=\\"32px\\" xml:space=\\"preserve\\" xmlns=\\"http://www.w3.org/2000/svg\\" xmlns:xlink=\\"http://www.w3.org/1999/xlink\\"> <path d=\\"M27.877,19.662c0.385-1.23,0.607-2.531,0.607-3.884c0-7.222-5.83-13.101-13.029-13.194v4.238    c4.863,0.093,8.793,4.071,8.793,8.956c0,0.678-0.088,1.332-0.232,1.966l-3.963-1.966l2.76,8.199l8.197-2.762L27.877,19.662z\\"></path> <path d=\\"M7.752,16.222c0-0.678,0.088-1.332,0.232-1.967l3.963,1.967l-2.76-8.199L0.99,10.785l3.133,1.553    c-0.384,1.23-0.607,2.531-0.607,3.885c0,7.223,5.83,13.1,13.03,13.194v-4.238C11.682,25.086,7.752,21.107,7.752,16.222z\\"></path> </svg>",
            }
          }
        />
      </a>
    </span>
  </div>
</div>
`;
